src/main/java/tech/ydb/io/r2dbc/query/R2dbcQueryParser.java

```diff
@@ -0,0 +1,8 @@
+package tech.ydb.io.r2dbc.query;
+
+/**
+ * @author kuleshovegor
+ */
+public interface R2dbcQueryParser {
+    YdbQuery parseYdbQuery(String sql);
+}
```

Comment:
Interfaces for a single class are not good.

Reply:
Made interface to class with static method.